models/unit/unit.go

```diff
@@ -193,6 +193,14 @@ func (u Unit) IsLessThan(unit Unit) bool {
 	return u.Idx < unit.Idx
 }
 
+// MaxPerm returns the max perms of this unit
+func (u Unit) MaxPerm() perm.AccessMode {
+	if u.Type == TypeExternalTracker || u.Type == TypeExternalWiki {
+		return perm.AccessModeRead
+	}
+	return perm.AccessModeAdmin
```

Comment:
```suggestion
	if u.Type != TypeExternalTracker && u.Type != TypeExternalWiki {
		return perm.AccessModeAdmin
	}
	return perm.AccessModeRead
```

Reply:
@noerw I think in this case it's not important ... but :point_up: that's how it would look like ... - I think both options are just fine

Reply:
It's the same.